ivy.el

```diff
@@ -189,13 +189,22 @@ Only \"./\" and \"../\" apply here.  They appear in reverse order."
   "Decide where to display the candidates.
 This function takes a string with the current matching candidates
 and has to display it somewhere.
-See https://github.com/abo-abo/swiper/wiki/ivy-display-function."
+See https://github.com/abo-abo/swiper/wiki/ivy-display-function.
+Note: this variable is used to config global display function,
+If user want to set a command's display function, for example:
+the display function used by ivy-completion-in-region, please
+use `ivy-display-functions-alist' instead."
```

Comment:
This is a good opportunity to improve the whole docstring:

```el
  "Determine where to display candidates.
When nil (the default), candidates are shown in the minibuffer.
Otherwise, this can be set to a function which takes a string
argument comprising the current matching candidates and displays
it somewhere.

This user option acts as a global default for Ivy-based
completion commands.  You can customize the display function on a
per-command basis via `ivy-display-functions-alist', which see.
See also URL
`https://github.com/abo-abo/swiper/wiki/ivy-display-function'."
```

Reply:
cool